plugins/postgres/dbt/adapters/postgres/impl.py

```diff
@@ -25,7 +25,10 @@ def date_function(cls):
 
     @available_raw
     def verify_database(self, database):
-        database = database.strip('"')
+        if database.startswith('"'):
```

Comment:
This isn't the most pleasant thing in the world, but I think it's a good and appropriate fix for this problem. Nice